docs/reference/modules/cluster/disk_allocator.asciidoc

```diff
@@ -123,7 +123,7 @@ Controls the flood stage watermark for dedicated frozen nodes, which defaults to
 `cluster.routing.allocation.disk.watermark.flood_stage.frozen.max_headroom` {ess-icon}::
 (<<dynamic-cluster-setting,Dynamic>>)
 Controls the max headroom for the flood stage watermark for dedicated frozen
-nodes. Defaults to 20GB when
+nodes. Defaults to 20gb when
```

Comment:
Hi @DaveCTurner , I actually did this because I noticed that ByteSizeValue.toString() outputs it as `gb` rather than `GB` or `GiB`. So currently when the max headroom is output, it's `20gb` rather than the default `20GB`.

But I do agree with you. Maybe I leave this as it was, and I create a new minor enhancement issue to improve how ByteSizeValue/ByteSizeUnit output strings? So that they ideally output `GiB` or `GB` instead of `gb` (similar for other units of course)?

Reply:
I think that this would be too large a change unfortunately, we've used lower-case byte-size units since forever (yes `mb` means megabytes not millibits). We probably could fix this given sufficient effort but it'd be a big breaking change so best left alone.

TBC `20gb` is fine, it's just an unformatted "20gb" that isn't.